test/app/export/about/index.html

```diff
@@ -0,0 +1,62 @@
+<!doctype>
```

Comment:
Missing ` html`?

Reply:
yep! good catch, thanks. unrelated to these test failures but a mistake nonetheless